ktor-client/ktor-client-features/ktor-client-auth/common/src/io/ktor/client/features/auth/Auth.kt

```diff
@@ -81,4 +82,5 @@ public fun HttpClientConfig<*>.Auth(block: Auth.() -> Unit) {
  * be removed after that.
  */
 @PublicAPICandidate("1.6.0")
+@ThreadLocal
```

Comment:
Since `AttrubuteKey` does not overwrite `equals `, it is important to have a single reference to this property. Please use `@SharedImmutable` instead.

Reply:
Thank you for the comment. I changed it to `@SharedImmutable`